cluster/gce/coreos/helper.sh

```diff
@@ -46,6 +46,11 @@ function create-master-instance() {
   local address_opt=""
   [[ -n ${1:-} ]] && address_opt="--address ${1}"
 
+  local preemptible_master=""
+  if [[ "${PREEMPTIBLE_MASTER}" == "true" ]]; then
```

Comment:
It doesn't expand the same way when PREEMPTIBLE_MASTER is unset. It causes an unbound variable error. If you were running ./cluster/kube-up.sh, it was probably always set because it's being set in config-gce.sh. What if I had my own custom config-blah.sh and I didn't update it with `PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}`? kube-up would be broken